common/src/View/EntityAttributeGridTable.cpp

```diff
@@ -511,6 +513,69 @@ namespace TrenchBroom {
             m_showDefaultRows = showDefaultRows;
             update();
         }
+        
+        static StringSet allSortedAttributeNames(MapDocumentSPtr document) {            
```

Comment:
This is a minor thing: I like to sort methods top down, i.e., I put the methods below their (bottom-most) usage in a class, esp. for private helper methods. The idea is that you can read a class top down.